src/hmem_cuda.c

```diff
@@ -758,7 +758,7 @@ enum cuda_xfer_setting cuda_get_xfer_setting(void)
 
 bool cuda_is_ipc_enabled(void)
 {
-	return !ofi_hmem_p2p_disabled() && cuda_ipc_enabled;
```

Comment:
this commit slipped in. Can you rebase on lastest main?

Reply:
done